api/routes.go

```diff
@@ -86,6 +86,8 @@ func addRoutes(r *gin.Engine, auth Authentication, tokenHandler TokenHandler, uc
 
 	router.GET("/scenario-publications", tom, handleListScenarioPublications(uc))
 	router.POST("/scenario-publications", tom, handleCreateScenarioPublication(uc))
+	router.POST("/scenarios-publications-backtest", tom,
```

Comment:
Hmm, ok here's an open though about publications.
Without discussing the details of how the publications log is currently stored (not necessarily in the most elegant way), the intention behind it is that we store a full history of what version was active from when to when, for audit purposes.
While that serves a purpose for "real live versions", I don't think that's necessary for "shadow test versions".
So perhaps we can skip the whole procedure of "storing a scenario publication", and as a consequence just use a usecase/endpoint pair that would be essentially called "start test mode"
What do you think ?